### PR TITLE
Add Validation Questions to Day 5

### PR DESCRIPTION
**Pull request by tracedwax**
I've asked one or both of these questions at the end of each sprint, to gauge user excitement about using the product the prototype could one day become.

It has the desired effect of determining whether user interest is strong or lukewarm. However, the growth hackers who created these questions to evaluate deployed products, and prototypes may be too preliminary. I still like to ask the questions anyway, to get a high level of validation before proceeding to a build. What are your thoughts?

### CODE REVIEW

File: 5-Test/README.md [hunk 1]
```diff
@@ -10,13 +10,43 @@ Test table](../Exercises/assumptions.md)
 
 ## Activities for Validate and Learn
 
-#### Gorilla Usability Testing
+#### Guerilla Usability Testing
```
Comment (kylefiedler) [inline 5-Test/README.md]: spelling

File: 5-Test/README.md [hunk 2]
```diff
@@ -10,13 +10,43 @@ Test table](../Exercises/assumptions.md)
...
+colleagues?”
+
+Before asking this, if the prototype didn’t include a scenario appropriate to
+the participant’s own use case, frame the question in terms of that use case
+first. Sometimes users and use cases differ and it’s not possible to customize
+the prototype for each participant. As an extreme example, if you prototyped an
+app for dogs but the participant was a cat owner instead, say “imagine this was
+an app about cats instead of dogs. On a scale of 0-10,...”
+
+Note that the answer to this question may not be what you want to hear. Design
```
Comment (kylefiedler) [inline 5-Test/README.md]: I'm not sure we need this disclaimer here. Throughout the sprint they should be willing to hear things that they might not want to hear.

**Comment by kylefiedler** [comment]
I personally don't like asking these questions because we're asking them to make assumptions. I trust users assumptions as much as I trust my own spelling ability. Which is to say I don't. It's also easy to word the questions so that they are leading, maybe link to Paul's [What not to Ask](http://robots.thoughtbot.com/what-not-to-ask).

**Comment by tysongach** [comment]
@tracedwax @kylefiedler Is there an update or action here?

**Comment by kylefiedler** [comment]
We should probably change Gorilla. I think the rest of this is fairly leading and probably shouldn't be added. There is the opportunity to add a section about questions though.

**Comment by tysongach** [comment]
Closing; [this repo has been deprecated](https://github.com/thoughtbot/design-sprint/commit/cdf3f3802adb58033ede2a7c2c45fe97e5601e51).

**Pull request closed without merging**